### DISCUSSION TITLE
General guide to creating reproduction carts?

### DISCUSSION THREAD

**Discussion by Dinierto** [sanni/cartreader#1051] [General Chat]
Hi there!  A friend and I are both looking to get into making some reproduction carts for hacks, etc. and I assume this device is a good place to start?

Are there any guides that walk you through what types of boards you need for each system and type of game?  Google really seems to be coming up dry, maybe I'm searching for the wrong thing.  Looking for a general direction to look as to what we need to buy and do.

Thanks!

**Comment by StanMOI** [comment]
> Hi there! A friend and I are both looking to get into making some reproduction carts for hacks, etc. and I assume this device is a good place to start?
> 
> Are there any guides that walk you through what types of boards you need for each system and type of game? Google really seems to be coming up dry, maybe I'm searching for the wrong thing. Looking for a general direction to look as to what we need to buy and do.
> 
> Thanks!

My first functional repro took more than 2 years.
I think no one will give you that kind of information.
Google, read and learn, try , test, re-test etc. This is the only way.
On Google You can find many open source ready projects that can help you to learn or to speculate.
Goodluck 

Here is my PCB "guides" for my private SNES reproduction :
[https://drive.google.com/drive/folders/1JY5F6gdS84obD6Sq_c2mIFIaMhnbhUZR?usp=drive_link](https://drive.google.com/drive/folders/1JY5F6gdS84obD6Sq_c2mIFIaMhnbhUZR?usp=drive_link)
![FLASH SNES KITAI4 (REV3) FRONTSIDE 3D ALL COMPONENTS @stanmoi25](https://github.com/user-attachments/assets/01ba7717-1248-4be4-b9a4-33528cca6808)



[https://drive.google.com/drive/folders/1_-ChMaNYtjCx3njl40RDagAwKr155d7Y?usp=sharing](https://drive.google.com/drive/folders/1_-ChMaNYtjCx3njl40RDagAwKr155d7Y?usp=sharing)
<img width="688" alt="SNES Repro 8-16-32MBit frontside @stanmoi25 copie" src="https://github.com/user-attachments/assets/a3593888-130e-419f-b443-6e159f011556" />


[https://drive.google.com/drive/folders/11hyGwWacszPCwKn91sqsfH50kHCUICx5?usp=sharing](https://drive.google.com/drive/folders/11hyGwWacszPCwKn91sqsfH50kHCUICx5?usp=sharing)
<img width="735" alt="SNES Repro 4-8mbit @stamoi25" src="https://github.com/user-attachments/assets/630a8d1a-0f5a-48e2-b8ae-2f3328970e14" />


[https://drive.google.com/drive/folders/1--1QgTp8I77WxKTw0IXoJbvjeQ9WhQ6t?usp=sharing](https://drive.google.com/drive/folders/1--1QgTp8I77WxKTw0IXoJbvjeQ9WhQ6t?usp=sharing)
![SNES Repro PCB 8-64mbit stanmoi25 copie](https://github.com/user-attachments/assets/595baa3f-2bd8-4bed-b8da-9b5d6235460e)

**Reply by Dinierto** [reply]
Oh wow thank you!

**Comment by MouseBiteLabs** [comment]
I have open source carts for the OSCR here:

https://github.com/MouseBiteLabs/Super-Nintendo-Cartridges

And my website has a guide for making other reproductions here:

https://mousebitelabs.com/category/snes-reproductions/

The boards on my website are currently closed source but I am working to open source them soon, after I finish the Game Boy boards I'm working on.
